package.json

```diff
@@ -41,7 +41,7 @@
     "postcss": "^8.4.5",
     "postcss-calc": "^8.2.2",
     "prettier": "^2.5.1",
-    "puppeteer": "^13.0.1",
+    "puppeteer": "^13.1.1",
```

Comment:
I noticed that Puppeteer 13.1.2 is released now, please see https://github.com/puppeteer/puppeteer/releases, and while I don't think that will fix our Windows bot issues it can't hurt to use the newer version while we're at it.